PocketKit/Sources/PocketKit/MyList/ArchivedItemsList/ArchivedItemsListViewModel.swift

```diff
@@ -432,3 +432,30 @@ extension ArchivedItemsListViewModel {
         return snapshot
     }
 }
+
+// MARK: - Clearing presented content
+extension ArchivedItemsListViewModel {
+    func clearSharedActivity() {
+        sharedActivity = nil
+        selectedItem?.clearSharedActivity()
+    }
+
+    func clearPresentedWebReaderURL() {
+        switch selectedItem {
+        case .readable(let readable):
+            readable?.clearPresentedWebReaderURL()
+        case .webView:
+            selectedItem = nil
```

Comment:
why do we do this for archived items, but not for saved?

Reply:
yep, that's an oversight on my part. Thanks!